content/03.methods.md

```diff
@@ -19,5 +19,18 @@
 ### Gene Expression Abundance Estimation
 
 ### RNA Fusion Calling and Prioritization
+**Gene Fusion Detection**
+> Arriba v1.1.0 and STAR-Fusion 1.5.0 fusion detection tools were set up using CWL on CAVATICA.
+> Aligned bam and chimeric sam files from STAR were used as inputs and GRCh38_gencode_v27 GTF was used for gene annotation in both tools.
+> STAR-fusion was run with default paramters and fusion calls were annotated with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz. 
+> While running arriba, the additional blacklist file blacklist_hg38_GRCh38_2018-11-04.tsv.gz was provided in the arriba release tarballs to remove recurrent fusion artifacts and transcripts present in healthy tissue.
+> Additionally strandedness information was provided to arriba or set to auto-detection for polyA samples.
+
+**Fusion prioritization**
+> All inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed at TPM >1 were used as true calls.
+> If a fusion call had large number of spanning fragment reads compared to junction reads (if spanning fragment - junction read > 10) or if either 5\` or 3\` genes multi-fused > 5 different genes these were removed as a potential false positive.
+> A list of curated list of fusion calls for each histology was used to capture each occurence of the fusion as a putative driver fusion.
```

Comment:
Where was this obtained from?

Reply:
@kgaonkar6 - can you check whether all of these canonical fusions still remain if we do not use this list (ie they should be in the oncogene/TSG/COSMIC, etc lists)? If so, we can remove this list and part of the code.